__exekutir__/roles/kommandir_workspace_setup/tasks/main.yml

```diff
@@ -123,8 +123,9 @@
   git:
     repo: "{{ item.repo }}"
     dest: "{{ kommandir_workspace }}/cache/{{ item.cachepath }}"
-    # Shallow clone is much faster, two commits will cover a merge-commit
-    depth: "{{ item.depth | default(2) }}"
+    # Shallow clone is much faster, two commits will cover a merge-commit and will be set per repo.
+    # default(omit) to workaround failure with python-bugzilla repo
+    depth: "{{ item.depth | default(omit) }}"
```

Comment:
This makes me uncomfortable. It adds too much dependency on kommandir.yml, and in particular, requiring that the maintainer magically know to add 'depth: 2' in some places but not if there's a specific tag.

Since the reason for this change is (IIUC): "do a shallow copy if we can, but we can't do that for tagged versions", how about:

    depth: "{{ omit if item.version is defined else item.depth | default(2) }}"   **untested**

I haven't tested this, and am not sure how to do so, and am particularly concerned about the `|` because I don't know how it combines with `omit`. If that fails, maybe this could work:

    depth: "{{ item.depth | default(omit if item.version is defined else 2) }}"

The point is, this is a one-time edit that will do the right thing and never cause problems when, in six months, someone adds a new repo with a version and a depth.